lib/config/presets/internal/config.ts

```diff
@@ -3,7 +3,15 @@ import type { Preset } from '../types';
 /* eslint sort-keys: ["error", "asc", {caseSensitive: false, natural: true}] */
 
 export const presets: Record<string, Preset> = {
-  base: {
+  'js-app': {
+    description: 'Default configuration for webapps.',
+    extends: ['config:recommended', ':pinAllExceptPeerDependencies'],
+  },
+  'js-lib': {
+    description: 'Default configuration for libraries.',
+    extends: ['config:recommended', ':pinOnlyDevDependencies'],
+  },
+  recommended: {
     description: 'Default base configuration for all languages.',
```

Comment:
```suggestion
    description: 'Recommended configuration for most users. It does not matter what programming language you use.',
```

Is this the correct place to edit the `description` that we show in our docs?

I think we should explain to our users that the `config:recommended` preset is the "basic" preset that the maintainers of Renovate "recommend" to most users, no matter what programming language they use.

By saying `most users` we leave the door open to recommend the `config:best-practice` preset as the ultimate best thing to do. 😉